tbx/static_src/sass/components/_team-listing.scss

```diff
@@ -0,0 +1,61 @@
+@use 'config' as *;
+
+.team-listing {
+    display: flex;
+    flex-wrap: wrap;
+    column-gap: $spacer-small;
+    row-gap: $spacer-small-plus;
+
+    @include media-query(large) {
+        max-width: 1270px;
+    }
+
+    &__item {
+        width: 370px;
+
+        @include media-query(medium) {
+            width: calc(33% - #{$spacer-small});
+        }
+
+        @include media-query(large) {
+            width: 230px;
+        }
+    }
+
+    &__image {
+        width: 100%;
+        height: 374px;
```

Comment:
This means the images aren't square on mobile - just checking that's intentional?

With:

![image](https://github.com/torchbox/torchbox.com/assets/31627284/af9354d0-0b42-4c98-9af8-d33f385aff82)

Without:

![image](https://github.com/torchbox/torchbox.com/assets/31627284/64bbdb19-902d-40c0-9136-75a16ec37ea1)

Reply:
The designs had the height slightly different to the width at mobile. I'll check again as it is maybe too tall at the smallest mobile screensize.